index.md

```diff
@@ -8,45 +8,99 @@ I offer a series of seminars designed to help scientists, engineers and other do
 
 **If your company has domain experts who are writing code, [get in touch](mailto:hi@tkiley.co.uk) today to increase both the reliability of the software you produce, as well as the speed at which you can develop it.**
 
-You can read more about what the course covers and how it is structured [here](course.html). 
+I have been writing code for ten years in a range of industries including science, technology, games and healthcare. In this time I have learnt the importance of good software practises like writing automated tests to check the code is correct and using continuous integration.
```

Comment:
s/like writing automated tests to check the code is correct and using continuous integration//

Unnecessary technical details in the intro.